src/components/monitor-list/monitor-list.jsx

```diff
@@ -8,15 +8,32 @@ import {stageSizeToTransform} from '../../lib/screen-utils';
 
 import styles from './monitor-list.css';
 
+// Use static `monitor-overlay` class for bounds of draggables
+// This is just a dummy div which doesn't scale to make the
+// bounds tracking play nicer with react-draggable
+const MonitorOverlay = () => (
+    <div
+        className="monitor-overlay"
+        style={{
+            width: 480,
+            height: 360,
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            visibility: 'hidden'
+        }}
+    />
+);
+
```

Comment:
react-draggable really did not like using a scaled container as the bounding reference, but we needed the scaling both visually as well as for proper drag deltas for the monitors.  This dummy div of a constant width is a hacky solution that makes react-draggable happy.  A better solution would likely be way more involved, perhaps replacing react-draggable entirely.